src/frontend/js/modules/header/header.js

```diff
@@ -118,6 +118,18 @@
           scope.domainId = session.domain._id;
           scope.baseUrl = window.openpaas && window.openpaas.OPENPAAS_API_URL || '';
 
+          (function() {
+            esnRestangular.one('themes', scope.domainId)
+              .withHttpConfig({ responseType: 'arraybuffer' })
+              .customGET('logo').then(function(response) {
+                const blob = new Blob(
+                  [response.data], { type: response.headers('Content-Type') }
+                );
+
+                scope.logoUrl = (window.URL || window.webkitURL).createObjectURL(blob);
+              });
+          })();
```

Comment:
Is there a reason to use a self called function here?

Reply:
Not necessary, It's more organisational